docs/multicluster/user-guide.md

```diff
@@ -142,10 +142,15 @@ We recommend creating one ServiceAccount for each member for fine-grained access
        namespace: antrea-mcs-ns
    ```
 
-2. Copy the access token into the member cluster `test-cluster-east`. E.g.
+2. Generate the access token file from leader cluster, and apply it in the member cluster `test-cluster-east`, e.g.
 
    ```bash
+   # Generate the file 'member-east-access-token.yml' from your leader cluster
+   export KUBECONFIG=/path/to/kubeconfig-of-leader-cluster
```

Comment:
Better not to export KUBECONFIG. Could you think about one of the following:

1. KUBECONFIG=/path/to/kubeconfig-of-leader-cluster && \
2. Use kubectl context
3. Use --kubeconfig

Reply:
Probably we should make fetching token a separate step.